Makefile

```diff
@@ -113,6 +113,47 @@ distclean:
 
 check: test
 
+# Remove files generated by running coverage, put the un-intstrumented lib back in place 
+clean_coverage:
```

Comment:
nit: This can be `clean-coverage`, all other targets in this file also use hyphens